draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1236,6 +1236,11 @@ If the certificate is logged, an SCT MUST be issued. Logging the certificate is
 useful, because monitors ({{monitor}}) can then detect these encoding errors,
 which may be accepted by some TLS clients.
 
+If `submission` is an accepted trust anchor whose certifier is neither an
+accepted trust anchor nor the first element of `chain`, then the log MUST return
```

Comment:
I'm still confused by this bit: "... the first element of `chain`":
When is it valid to submit an accepted trust anchor?
It seems to me the answer is:
- When there's another trust anchor that certifies it, or:
- When there's a chain that leads to another trust anchor.
Is that right? Are self-signed trust anchors considered as submissions "whose certifier is a trust anchor" - itself?